src/components/attendee/EventInfo/EventInfo.tsx

```diff
@@ -0,0 +1,50 @@
+import ShowMoreText from "react-show-more-text";
+import cn from "classnames";
+
+import { STRING_DASH_SPACE, STRING_SPACE } from "settings";
+
+import { WorldEvent } from "types/venues";
+
+import { eventEndTime, eventStartTime } from "utils/event";
+import { formatTimeLocalised } from "utils/time";
+
+import { RenderMarkdown } from "components/organisms/RenderMarkdown";
+
+import CN from "./EventInfo.module.scss";
+
+interface EventInfoProps {
+  event: WorldEvent;
+  centered?: boolean;
+}
+
+const moreLessSpan = (text: string) => (
+  <span className={CN.moreLessSpan}>{text}</span>
+);
+
+export const EventInfo: React.FC<EventInfoProps> = ({ event, centered }) => (
+  <div data-bem="EventInfo" className={cn(CN.general, centered && CN.centered)}>
```

Comment:
[Question] Can we transform this to
```suggestion
  const generalClassnames = cn(CN.general, {
  [CN.centered]: centered
  })
```
I thought we've agreed on this variant for classnames usage. It gets a little tough to see the resulting classnames when they're calculated inside the markup

Reply:
done: https://github.com/sparkletown/sparkle/pull/3071/commits/53ad693cb533a0eaa1fddc004d2ab374630c2be1